### PR TITLE
Added support for actions on multiple processes

### PR DESCRIPTION
**Pull request by plonibarploni**
From gnome-system-monitor.

Note that there is a critical error that shows up in the console, however it performs as expected:
`Gtk-CRITICAL: gtk_tree_selection_count_selected_rows: assertion 'GTK_IS_TREE_SELECTION (selection)' failed`

### CODE REVIEW

**Comment by rbuj** [comment]
Remove the critical error (test `data->selection` is not `NULL`):
```patch
diff --git a/src/interface.cpp b/src/interface.cpp
index 08e40e5..3d259d2 100644
--- a/src/interface.cpp
+++ b/src/interface.cpp
@@ -785,7 +785,7 @@ update_sensitivity(ProcData *data)
     GtkAction *action;
 
     processes_sensitivity = (data->config.current_tab == PROCMAN_TAB_PROCESSES);
-    selected_sensitivity = (processes_sensitivity && gtk_tree_selection_count_selected_rows (data->selection) > 0);
+    selected_sensitivity = (processes_sensitivity && data->selection && gtk_tree_selection_count_selected_rows (data->selection) > 0);
 
     if(data->endprocessbutton) {
         /* avoid error on startup if endprocessbutton
```

File: src/procdialogs.cpp [hunk 1]
```diff
@@ -164,11 +184,21 @@ procdialog_create_renice_dialog (ProcData *procdata)
...
     if (!info)
         return;
 
-    dialog_title = g_strdup_printf (_("Change Priority of Process “%s” (PID: %u)"),
-                                    info->name, info->pid);
+    if ( selected_count == 1 ) {
+        dialog_title = g_strdup_printf (_("Change Priority of Process “%s” (PID: %u)"),
+                                        info->name, info->pid);
+    } else {
+        dialog_title = g_strdup_printf (_("Change Priority of %d selected processes"),
```
Comment (rbuj) [inline src/procdialogs.cpp:198]: Use title case, .i.e `Change Priority of %d Selected Processes`

**Comment by sc0w** [comment]
@plonibarploni 

which is the gnome-system-monitor commit(s)?

**Comment by plonibarploni** [comment]
> which is the gnome-system-monitor commit(s)?

I based it on https://github.com/GNOME/gnome-system-monitor/commit/5cf1225d86929114d6f0424112fd2abc8880dc26

File: src/procdialogs.cpp (hunk 1 above)
Comment (plonibarploni) [reply src/procdialogs.cpp]: done

**Comment by raveit65** [comment]
@plonibarploni 
Can you please change commit message?
Something like this.
```
Added support for actions on multiple processes
<blank line>
ported from gnome-system-monitor:
https://github.com/GNOME/gnome-system-monitor/commit/5cf1225d86929114d6f0424112fd2abc8880dc26
```
No need to write in commit header that you ported the commit ;)
In commit header should only be written with a short description what the commit does.
Additional infos and references belongs in commit body.

**Comment by raveit65** [comment]
Ok, it works when choosing the action from menu, but not with choosing the action from context menu (right-click) for several selected processes.
I think this isn't related to PR, it seems for this is some extra work needed.

**Pull request merged**